File: src/net/BukkitPE/level/particle/LargeExplodeParticle.java
package net.BukkitPE.level.particle;

import net.BukkitPE.math.Vector3;

/**
 * Created on 2015/11/21 by xtypr.
 * Package net.BukkitPE.level.particle in project BukkitPE .
 */
public class LargeExplodeParticle extends GenericParticle {
    public LargeExplodeParticle(Vector3 pos) {
        super(pos, Particle.TYPE_LARGE_EXPLODE);
    }
}

File: src/BukkitPE/level/particle/LargeExplodeParticle.php
<?php
namespace BukkitPE\level\particle;

use BukkitPE\math\Vector3;

class LargeExplodeParticle extends GenericParticle{
	public function __construct(Vector3 $pos){
		parent::__construct($pos, Particle::TYPE_LARGE_EXPLODE);
	}
}
